### ISSUE TITLE
In some cases mixin injection throws "org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 257: Insufficient maximum stack size."

### ISSUE THREAD

**Issue by Danklor** [FabricMC/Mixin#31]
There isn't only one mod or set of mods that triggers this issue. I can disable some mods to make the error stop, but enabling different mods instead also triggers the issue. I do not know if there is anything in common to the various sets of mods that will trigger the problem. The exception is not fatal and Minecraft does continue to load and run (although I suspect with some missing mixins.)

Full log attached. I would be glad to provide additional logs of different mod combinations if that would be helpful.

OS: MacOS 10.14.6
Java: Oracle 1.8.0_251 64-bit
Launcher: MultiMC 0.6.11-1370
Fabric: 0.8.9+build.203

[Insufficient_stack.txt](https://github.com/FabricMC/Mixin/files/4898741/Insufficient_stack.txt)

**Comment by Chocohead** [comment]
I don't think you could ever cause this with a single mod, except possibly if the single mod injected into the same method twice. A specific pair of mods I thought would so long as they loaded in a certain (wrong) way. Either way, different mod combinations would be very helpful, but above all else a small a set as possible would be ideal.

For that crash report at least Patchouli's crash handler is (statically) loading some things drastically out of order but is just a coincidental victim as far as I can tell. Some chain loaded class from `Minecraft` (almost certainly rather than `Screen`) will have some combinations of Mixins which are causing problems together (but not alone in their own right).

**Comment by Danklor** [comment]
After many mod combinations I found that the one common mod in all cases was appleskin-mc1.16-fabric-1.0.11. Without that mod loaded the stack issue does not occur.  I will continue to find the least mods (including AppleSkin) needed to trigger the problem.

[Works_without_AppleSkin.txt](https://github.com/FabricMC/Mixin/files/4900032/Works_without_AppleSkin.txt)

**Comment by Danklor** [comment]
Here's the log of the most pared down set of mods I could make fail. This still has 41 enabled items in the mods folder so it does take quite a few mixins to run out of stack space.  If any one of these mods are disabled then the stack does not run out. One thing of note is that I did not include Patchouli in this mod set.

[Insufficient_stack_2.txt](https://github.com/FabricMC/Mixin/files/4900860/Insufficient_stack_2.txt)
